TykTechnologies/tyk
# [TT-10370] Alias imports

This PR aliases imports within gateway packages, so the aliases are non-conflicting inside individual package scopes.

To verify, run the following:

```
go install github.com/TykTechnologies/exp/cmd/go-fsck@978afcf81429325b9dcf51ec2afcc77a3d83e338
```

```
go-fsck lint ; echo $?
```

The command should exit with status 0 printed in the last line; The command prints a few warnings around ignored files due to build tags. This is expected and can be ignored.

https://tyktech.atlassian.net/browse/TT-10370
